packages/features/bookings/Booker/Booker.tsx

```diff
@@ -37,15 +37,19 @@ const useBrandColors = ({ brandColor, darkBrandColor }: { brandColor?: string; d
   useCalcomTheme(brandTheme);
 };
 
-const BookerComponent = ({ username, eventSlug, month, rescheduleBooking }: BookerProps) => {
+const BookerComponent = ({ username, eventSlug, month, rescheduleBooking, hideBranding }: BookerProps) => {
   const { t } = useLocale();
+  const { data: event, isLoading } = trpc.viewer.public.event.useQuery(
```

Comment:
had to use the vanilla trpc query rather than the `useEvent` hook because prefetched data in the `serverSideProps` isn't reused here. maybe because of some race conditions with zustand , `useInitializeBookerStore` is called after the `useEvent` hook and both depends on each other

Reply:
The reason the new booker uses the `useEvent` hook is because it should do everything client side rather than server side. Soon the booker component will get published to NPM, and should then be able to run fully standalone, so without any serverside props passed in. Therefore we actually have to revert this change.